editions/tw5.com/tiddlers/plugins/Plugin Ordering.tid

```diff
@@ -0,0 +1,51 @@
+created: 20220613115453346
+modified: 20220613171815188
+tags: PluginMechanism
+title: Plugin Ordering
+
+Using the Node.js client-server configuration, plugins are activated in the following order:
+
+# Plugins found using the OS environment variables
+#* And: [[PluginFolders]]
+#* See: [[Environment Variables on Node.js]]
+
+# Plugins stored in the wiki `/plugin` path
+#* See: [[PluginFolders]]
+
+# Plugins specified in the command line
+#* See: [[Using TiddlyWiki on Node.js]] ... the notes about <<.from-version "5.1.20">>
+
+# Plugins imported by drag & drop as wiki content
+#* See: [[Installing a plugin from the plugin library]] or [[Manually installing a plugin]]
+
+''Important:''
+
+<<<
```

Comment:
It looks like you're using the quote syntax for emphasis, and not for an actual quote. It would be better to avoid that, and reserve quotes for their intended semantic purpose.